### DISCUSSION TITLE
Not on Router WLAN Client List

### DISCUSSION THREAD

**Discussion by drgkt** [arendst/Tasmota#19651] [Q&A]
Hi
I have 2 smart plugs one of which is not shown on router wifi client list.
If I restart the router, it shows, then disappears again, however, it is  reachable via web interface. If I ping it, it appears on the list BUT with IPv4 address 0.0.0.0, instead of the actual one.
What commands you recommend I issue to see what the difference is between the 2 plugs and why this is happening?

EDIT: Both are under DHCP binding in router settings, so they get the same IP every time.

Thanks

**Comment by Jason2866** [comment]
Which brand is the router?

**Reply by drgkt** [reply]
ZTE ZXHN H268N.
But why one of them and not both?

**Reply by Jason2866** [reply]
Maybe the one is "just" defect.

**Reply by drgkt** [reply]
Maybe.
Or there are differences in configuration that I need to pinpoint...

**Comment by drgkt** [comment]
@barbudor
Suggestions?

**Reply by barbudor** [reply]
When unsure of the actual configuration, I'll do a 'reset 1' to wipe it out and restart from scratch.
